pdns/dnsdist.hh

```diff
@@ -262,7 +262,75 @@ struct DNSDistStats
   };
 };
 
+// Keeps additional information about metrics
+struct MetricDefinition {
+  MetricDefinition(std::string description, std::string prometheusType) {
```

Comment:
It doesn't matter much but it could take const references to string instead of copying the strings. The fields could also be initialized in the member initializer list.

Reply:
Reworked in previous patch set.